src/components/Sum/Sum.jsx

```diff
@@ -1 +1,20 @@
 // export const Sum = () => ();
+import React from 'react';
+
+export const Sum = ({ a = 0, b = 0 }) => {
+  const sum = a + b;
+
+  return (
+    <p>
+      Sum of
+      {' '}
+      {a}
+      {' '}
+      and
+      {b}
+      {' '}
+      is
+      {sum}
```

Comment:
we can rewrite it in one line using [Template literals (Template strings)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)